Libplanet/Store/IStore.cs

```diff
@@ -231,6 +231,42 @@ void PutBlock<T>(Block<T> block)
         /// <seealso cref="PutTxExecution(Libplanet.Tx.TxSuccess)"/>
         TxExecution GetTxExecution(BlockHash blockHash, TxId txid);
 
+        /// <summary>
+        /// Records a index for given pair <paramref name="txId"/> and <paramref name="blockHash"/>.
+        /// If there exist a record for <paramref name="txId"/> already,
+        /// it overwrites the record silently.
+        /// </summary>
+        /// <param name="chainId">The chain ID of the index.</param>
+        /// <param name="txId">The <see cref="TxId"/> of the <see cref="Transaction{T}"/>.</param>
+        /// <param name="blockHash">The <see cref="BlockHash"/> of the <see cref="Block{T}"/>.
+        /// </param>
+        public void PutTxIdBlockHashIndex(Guid chainId, TxId txId, BlockHash blockHash);
```

Comment:
The visibility keyword `public` is unnecessary in this case.

```suggestion
        void PutTxIdBlockHashIndex(Guid chainId, TxId txId, BlockHash blockHash);
```